modules/apps/ccv/types/expected_keepers.go

```diff
@@ -0,0 +1,17 @@
+package types
+
+// StakingKeeper defines the contract expected by parent-chain ccv module.
+// StakingKeeper is responsible for keeping track of latest validator set of all baby chains
+type StakingKeeper interface {
+	GetValidatorSetChanges(chainID string) []abci.ValidatorUpdate
+	// This method is not required by CCV module explicitly but necessary for init protocol
+	GetInitialValidatorSet(chainID string) []sdk.Tx
```

Comment:
why is this a tx slice instead of a Valset type? How do we represent validator sets for non-Tendermint chains on the leader chain? Should this also return the self height at which the validator set for the follower chain was initialized on the leader chain (baby = follower, parent = leader)

Reply:
This will be a list of genesis transactions that can be queried from the parent chain.

They can then be collected into the genesis file of baby chain to populate all staking genesis info. This is part of the init protocol that I have roughly sketched out above but yet to fully specify

Reply:
validators can be initialized without a genesis transaction. They could be prespecified in the staking section of the genesis file